tests/cases/standalone/common/mysql.result

```diff
@@ -0,0 +1,21 @@
+-- SQLNESS PROTOCOL MYSQL
+SHOW @@tx_isolation;
```

Comment:
sorry, it was my mistake the statement should `select @@tx_isolation` and `select @@version_comment`

Reply:
It's ok. I've replace the statement